src/hearings/models/hearingDetails.model.ts

```diff
@@ -19,4 +21,6 @@ export interface HearingDetailsModel {
   hearingRequester?: string;
   leadJudgeContractType?: string;
   totalParticipantAttendingHearing?: number;
+  cancelationReason: RefDataModel[];
+  cancelledCaseHearing: HearingListModel;
```

Comment:
this cancelationReason should be an independent payload set in the DELETE method, this should be decoupled with the existing model.